src/playground/public/assets/js/app.ts

```diff
@@ -107,3 +113,11 @@ function removeFadeInClass() {
   const resultElem = document.getElementById('result')!;
   resultElem.classList.remove('fadeIn');
 }
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
this function is used so the comment shouldn't be needed

Reply:
you are right. remove it